src/Nethermind.Int256/UInt256.cs

```diff
@@ -660,7 +660,20 @@ internal static ulong Rsh(ulong a, int n)
             return (a >> n1) >> n2;
         }
 
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        internal static ulong NativeLsh(ulong a, int n)
+        {
+            return a << n;
+        }
+
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        internal static ulong NativeRsh(ulong a, int n)
+        {
+            return a >> n;
```

Comment:
Could we `Debug.Assert` the requirement when this method can be called?

Reply:
Sure! Done in 2def59d